rosidl_typesupport_opensplice_cpp/CMakeLists.txt

```diff
@@ -23,6 +27,17 @@ ament_export_include_directories(include include/${PROJECT_NAME}/impl)
 
 ament_python_install_package(${PROJECT_NAME})
 
+add_library(${PROJECT_NAME} SHARED src/identifier.cpp)
```

Comment:
Would it be possible to roll this `identifier.cpp` into the shared library created in the `rmw_opensplice_cpp` package?:

https://github.com/ros2/rmw_opensplice/blob/master/rmw_opensplice_cpp/CMakeLists.txt#L24

As far as I know, unlike the `rosidl_typesupport_introspection` implementation, this `rosidl_` package is always paired to the matching `rmw_` package. The combination is always `user exec = user object files + librmw_opensplice_cpp.so + libmsg_pkg1_opensplice.so + libmsg_pkgN_opensplice.so` and now we're adding ` + librosidl_typesupport_opensplice_cpp.so` for just the identifier symbol. It would be great to simplify it to roll this symbol into the `rmw_opensplice_cpp` shared library, but I guess there might be use cases where that won't work?